.github/workflows/windows-macos.yml

```diff
@@ -332,8 +333,8 @@ jobs:
       shell: bash
       run: |
         NAME="${{ steps.cmake-package.outputs.name }}"
-        ARTIFACT_NAME="${{ matrix.runs-on }}-${{ matrix.arch }}-package-artifact-name=${{ matrix.runs-on }}-${NAME}"
-        FILE_NAME="${{ matrix.runs-on }}-${{ matrix.arch }}-package-file-name=${NAME}"
+        ARTIFACT_NAME="${{ matrix.runs-on }}-package-artifact-name=${{ matrix.runs-on }}-${NAME}"
+        FILE_NAME="${{ matrix.runs-on }}-package-file-name=${NAME}"
```

Comment:
I believe we need to keep the arch here. Packages for the two architectures need to be combined later, but this prevents that: https://github.com/canonical/multipass/actions/runs/18712914069/job/53371065345?pr=4467